src/lib/draw/SplitRouteMode.svelte

```diff
@@ -75,7 +82,10 @@
       // We clicked the map, stop the tool
       changeMode("edit-attribute");
     } else {
-      let result = lineSplit($gjScheme.features[snappedIndex], cursor);
+      let result = lineSplit(
+        $gjScheme.features[snappedIndex] as Feature<LineString>,
```

Comment:
Static analysis doesn't know we picked a linestring (vs a point or polygon), so we assert the type here